server/app/models/Product.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-underscore-dangle */
+const mongoose = require('mongoose');
+
+const options = {
+  timestamps: true,
+};
+
+const ProductSchema = mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: [true, 'Please add the product name'],
+    },
+    link: {
+      type: String,
+      required: [true, 'Please add the product link'],
+    },
+    imageUrl: {
+      type: String,
+      default: '',
```

Comment:
@aeciorc brought this up with me. Do we want to have a default image until the user sets one? Would we default it here or on the frontend?

Reply:
We can have the frontend show a default image if the string is empty. The default image can reside inside the /public/assets directory.